### ISSUE TITLE
visNetwork htmlwidget are not displayed

### ISSUE THREAD

**Issue by tilltnet** [dirkschumacher/encryptedRmd#17]
Hi, this is a very useful package!
When I use it with a htmlwidget created with the visNetwork package the widget is not displayed in the browser when decrypted. Short example below.

    ---
    title: "test_enrcrypytedRmd"
    output: encryptedRmd::encrypted_html_document
    ---
    
    ```{r setup, include=FALSE}
    knitr::opts_chunk$set(echo = TRUE)
    ```
    
    ```{r results='asis'}
    library(visNetwork)
    nodes <- data.frame(id = 1:3)
    edges <- data.frame(from = c(1,2), to = c(1,3))
    
    visNetwork(nodes, edges)
    ```

**Comment by dirkschumacher** [comment]
Thanks! Can you try the current master and see if that fixes it? Seems related to #10.

**Comment by tilltnet** [comment]
Tried it and it works!
Thank you so much!
